DEPRECATION.md

```diff
@@ -63,8 +63,9 @@ features:
 | Removed    | [MySQL-based SQL API](#mysql-based-sql-api)                                                                                       | v0.35.0    | v0.35.0   |
 | Removed    | [`initApp` hook](#initapp-hook)                                                                                                   | v0.35.0    | v0.35.0   |
 | Removed    | [`/v1/run-scheduled-refresh` REST API endpoint](#v1run-scheduled-refresh-rest-api-endpoint)                                       | v0.35.0    | v0.36.0   |
-| Deprecated | [Node.js 18](#nodejs-18)                                                                                                          | v0.36.0    |           |
-| Deprecated | [`CUBEJS_SCHEDULED_REFRESH_CONCURRENCY`](#cubejs_scheduled_refresh_concurrency) | v1.2.7 | |
+| Removed    | [Node.js 18](#nodejs-18)                                                                                                          | v0.36.0    | v1.3.0    |
```

Comment:
Then, we need sub-article for Node.js 18. We should write that it was removed.

<img width="971" alt="image" src="https://github.com/user-attachments/assets/9a1853c0-9a2b-48a6-92ab-8f00f23575ef" />